resources/accounts.py

```diff
@@ -44,9 +55,44 @@ def post(self):
                 return {'message': "Password is necessary to register"}, HTTPStatus.CONFLICT
 
             # create new account
-            new_account = AccountsModel(email=data['email'], username=data['username'])
+            new_account = AccountsModel(email=data['email'], username=data['username'], confirmed=False)
             # assign the hashed password to the user
             new_account.hash_password(data['password'])
+
+            # Generació del token de confirmacio de correu
+            email_token = self.generate_confirmation_token(data['email'])
+            print("confirmation token is: {}".format(email_token))
+
+            # confirm_url2 = flask.url_for('confirm', token=email_token, _external=True)
+            # print("URL2: ", confirm_url2)
+            # TODO: cambiar para coger url en funcion del entorno (local o cloud)
+            # 8080 for dev
+            confirm_url = "http://localhost:8080/#/emailConfirmation/validation_token=" + email_token
```

Comment:
El puerto debería de ser el 5000 para que funcione junto al resto de las peticiones
```suggestion
            confirm_url = "http://localhost:5000/#/emailConfirmation/validation_token=" + email_token
```